styles.css

```diff
@@ -107,19 +107,31 @@ h3 {
 /* SELECTORS */
 /***************************************************/
 /* Put Selector post-it styles here! ↓ */
+ dhivian-pirabaharan
+.post-it{
```

Comment:
The styling should be applied using your ID selector, not the post-it class: id="dhivian-p"